distro/install_script/default.nix

```diff
@@ -0,0 +1,41 @@
+{ disable-systemd ? "false", stage-2-hook ? "/bin/sh -l", log-level ? "error"
+, console ? "hvc0", test-command ? "", pkgs ? import <nixpkgs> { } }:
+let
+  aster-kernel = builtins.path {
+    name = "aster-nix-osdk-bin";
+    path = ../../target/osdk/iso_root/boot/aster-nix-osdk-bin;
+  };
+  etc-nixos = builtins.path { path = ../etc_nixos; };
+
+  aster_configuration = pkgs.replaceVarsWith {
+    src = ./aster_configuration_template.nix;
+    replacements = {
+      aster-kernel = aster-kernel;
+      aster-disable-systemd = disable-systemd;
+      aster-stage-2-hook = stage-2-hook;
+      aster-log-level = log-level;
+      aster-console = console;
+      aster-test-command = test-command;
+    };
+  };
+  install_aster_nixos = pkgs.replaceVarsWith {
+    src = ./install_nixos_template.sh;
+    replacements = {
+      aster-configuration = aster_configuration;
+      aster-etc-nixos = etc-nixos;
+    };
+    isExecutable = true;
+  };
+
+in pkgs.stdenv.mkDerivation {
+  name = "aster_nixos_installer";
```

Comment:
For consistency, rename the `install_script` directory to `aster_nixos_installer`.